generators/app/index.js

```diff
@@ -58,9 +65,18 @@ module.exports = yeoman.Base.extend({
 
     var prompts = [{
       type: 'input',
-      name: 'projectName',
+      name: 'repository',
+      message: 'Terra repository:',
+      default: 'terra-core',
+      validate: function (input) {
+        var validRepos = ['terra-core', 'terra-clinical'];
```

Comment:
Can we update the ReadMe for this generator to provide some guidance about what the user might expect when running it?